Documentation/concepts/networking.rst

```diff
@@ -189,3 +183,46 @@ this "ingress" load-balancing capability, which achieve the same effect that
 Cilium handles forwarding and security only for ''internal'' traffic between
 different services.
 
+.. _concepts_masquerading:
+
+Masquerading
+============
+
+As the IPv4 addresses used for pods are typically allocated from RFC1918
```

Comment:
I think it reads cleaner if we remove the condition in the beginning of the statement 
s/As the//